src/openarm/can/socket/openarm.cpp

```diff
@@ -31,6 +31,12 @@ OpenArm::OpenArm(const std::string& can_interface, bool enable_fd)
 void OpenArm::init_arm_motors(const std::vector<damiao_motor::MotorType>& motor_types,
                               const std::vector<uint32_t>& send_can_ids,
                               const std::vector<uint32_t>& recv_can_ids) {
+    
+    if (motor_types.size() != send_can_ids.size() ||
+        motor_types.size() != recv_can_ids.size()) {
+        throw std::invalid_argument(
+            "Motor types, send CAN IDs, and receive CAN IDs vectors must have the same size.");
```

Comment:
How about including actual sizes (`motor_types.size()`, `send_can_ids.size()` and `recv_can_ids.size()`) for easy to debug?